ActionGame/ActionGame/Script/Header/Player.h

```diff
@@ -1,20 +1,21 @@
+
 #ifndef PLAYER_H
 #define PLAYER_H
 
 class Player {
 public:
-	int pos_x;
-	int pos_y;
-	int movingSpeed;
-	int playerGHandle;
-
 	Player();
 	~Player();
 
 	// �ړ�
-	void Moving();
+	static void Moving();
 	// �`��
-	void Draw();
+	static void Draw();
+
+private:
+	static int pos_x;
+	static int pos_y;
+	static int movingSpeed;
```

Comment:
他に『速度』に関する情報は持っていないようなので、単純に
```suggestion
	static int speed;
```
で良いかと。

Reply:
@taishi-matsumura 
372a1b3 で修正しました。